tests/selftest_console.js

```diff
@@ -0,0 +1,68 @@
+const {closePage, newPage} = require('../browser_utils');
+
+async function run(config) {
+    const page = await newPage({...config, forward_console: true});
+    await page.evaluate(() => {
```

Comment:
Can we add a test here which reloads the page, e.g. via `page.setContent`, `page.reload` or `page.goto`? Without it, I'm not certain that we can install the hook with a simple `evaluate` rather than `evaluateOnNewDocument`

Reply:
Good catch! Changed it to `evaluateOnNewDocument`